inc/asset.php

```diff
@@ -33,12 +40,20 @@ class Optml_Asset extends Optml_Resource {
 	public function __construct( $url = '', $args = array(), $cache_buster = '' ) {
 		parent::__construct( $url, $cache_buster );
 
+		$settings = new Optml_Settings();
```

Comment:
we can have some static props on the class for the css/js minify status rather than getting them each time we do an asset process.